libs/ui-lib/src/lib/custom-file-upload/custom-file-upload.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ChangeEvent, ReactNode } from 'react';
```

Comment:
(sorry i couldn't leave this comment in app.scss)

Please remove
```scss
div {
  width: 100%;
}
```
from app.scss and fix the styling.